### PR TITLE
operator-sdk: update to v0.15.2

### PR DESCRIPTION
**Pull request by JAORMX**

### CODE REVIEW

**Comment by JAORMX** [comment]
/hold

Lets first merge #142

File: Makefile [hunk 1]
```diff
@@ -147,7 +147,7 @@ run: operator-sdk ## Run the compliance-operator locally
 	WATCH_NAMESPACE=$(NAMESPACE) \
 	KUBERNETES_CONFIG=$(KUBECONFIG) \
 	OPERATOR_NAME=compliance-operator \
-	$(GOPATH)/bin/operator-sdk up local --namespace $(NAMESPACE)
+	$(GOPATH)/bin/operator-sdk run local --namespace $(NAMESPACE)
```
Comment (jhrozek) [inline Makefile]: Looks like this change is not sufficient, you'll probably want to amend the line with this:
```
diff --git a/Makefile b/Makefile
index 0796b4d..58e4fbf 100644
--- a/Makefile
+++ b/Makefile
@@ -147,7 +147,7 @@ run: operator-sdk ## Run the compliance-operator locally
        WATCH_NAMESPACE=$(NAMESPACE) \
        KUBERNETES_CONFIG=$(KUBECONFIG) \
        OPERATOR_NAME=compliance-operator \
-       $(GOPATH)/bin/operator-sdk run local --namespace $(NAMESPACE)
+       $(GOPATH)/bin/operator-sdk run --local --namespace $(NAMESPACE)
 
 .PHONY: clean
 clean: clean-modcache clean-cache clean-output ## Clean the golang environment
```

Comment (JAORMX) [reply Makefile]: Fixed. Thanks!

**Comment by jhrozek** [comment]
thanks, make run now works.

/lgtm

**Pull request merged**